### DISCUSSION TITLE
Toooooooooooooooooõôöoooo slow

### DISCUSSION THREAD

**Discussion by b-y-f** [blue-yonder/tsfresh#967] [Q&A]
My 64M CSV data: 1,156,320 rows
MiniFeat work, sample work,

Efficient Not work, n_worker=64

**Comment by b-y-f** [comment]
n_workers=64 System crashed

**Reply by b-y-f** [reply]
it getting some process after 3-4 hours...

**Comment by nils-braun** [comment]
Not really a question (at least not one that I understand) or a discussion topic where other community members can contribute to. Therefore I am closing it now.
